ocaml/utils/language_extension.mli

```diff
@@ -49,13 +49,29 @@ with type 'a extn := 'a t
 (** Equality on language extensions *)
 val equal : 'a t -> 'b t -> bool
 
+(** The type of language extension universes. Each universe allows a set of
+    extensions, and every successive universe includes the previous one. *)
+module Universe : sig
+  type t =
+    | No_extensions
+    | Upstream_compatible
+        (** Upstream compatible extensions, also known as "erasable". *)
+    | Stable  (** Extensions of [Stable] maturity. *)
+    | Beta  (** Extensions of [Beta] maturity. *)
+    | Alpha  (** All extensions. Default. *)
```

Comment:
What does "Default" here refer to? Meaning "this is the universe enabled by default", or "this is the universe that an extension will belong to by default"?